lib/flux/app-state.js

```diff
@@ -449,6 +448,13 @@ var actionMap = new ActionMap( {
 					}
 
 					// working is the state of the note in the editor
+					const note = state.notes.find( n => n.id === noteId );
+
+					if ( ! note ) {
+						console.error( `Cannot find note (id=${id})!` );
```

Comment:
@dmsnell variable `id` is not defined, I think you meant `noteId`? maybe the `no-undef` rule should be added to `.eslintrc` to avoid this?